src/components/SeoForm/SeoForm.tsx

```diff
@@ -65,9 +78,13 @@ interface SeoFormProps {
   description?: string;
   descriptionPlaceholder: string;
   disabled?: boolean;
+  errors?: Array<PageErrorFragment | ProductErrorFragment>;
   loading?: boolean;
   helperText?: string;
+  isCreating?: boolean;
```

Comment:
I think the better name would be `allowEmptySlug`, it's more understandable from the component's perspective